### DISCUSSION TITLE
the published packages disappeared

### DISCUSSION THREAD

**Discussion by A-aronYang** [verdaccio/verdaccio#2389] [Q&A] [answered]
Sorry, I'm not a native speaker.
I have puslished one packge
![image](https://user-images.githubusercontent.com/52499709/130739129-4fde5ca2-1741-45a8-85f5-f150fe594ceb.png)

If I restart my linux , the packges I have published disappeared.
![image](https://user-images.githubusercontent.com/52499709/130738855-cb0a510e-ed7d-4d60-bd37-11abcf1dd364.png)

I don't know why it happend and how to config it
Please help me，If you see the question and thank you!

**Comment by juanpicado** [comment, marked as answer]
ref https://github.com/verdaccio/verdaccio/discussions/2441
